Directory.Packages.props

```diff
@@ -37,7 +37,8 @@
     <PackageVersion Include="Microsoft.Extensions.ObjectPool" Version="8.0.0" />
     <PackageVersion Include="Microsoft.Toolkit.Uwp.Notifications" Version="7.1.2" />
     <PackageVersion Include="Microsoft.Web.WebView2" Version="1.0.2088.41" />
-    <PackageVersion Include="Microsoft.Win32.SystemEvents" Version="8.0.0" /> <!-- Package added as a hack for being able to exclude the runtime assets so they don't conflict with 8.0.1. This is a dependency of System.Drawing.Common but the 8.0.1 version wasn't published to nuget. -->
+    <PackageVersion Include="Microsoft.Win32.SystemEvents" Version="8.0.0" />
+    <!-- Package added as a hack for being able to exclude the runtime assets so they don't conflict with 8.0.1. This is a dependency of System.Drawing.Common but the 8.0.1 version wasn't published to nuget. -->
```

Comment:
For like 40, the comment here should be above line 40, correct (and others).

if so, when updating it auto shifts to below so i can go in and flip.  Want to verify behavior.

Reply:
I'd prefer for it to stay in the the same line, but if the tool just automates it to below, I'd prefer for it to be on top, yes. I'll do the changes.